manifests/b/Bluesound/BluOSController/3.18.1.1675/Bluesound.BluOSController.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Bluesound.BluOSController
+PackageVersion: 3.18.1.1675
+PackageLocale: en-US
+Publisher: Bluesound
+PackageName: BluOS Controller
+License: Copyright (c) 2018. Lenbrook Industries Ltd

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate